kdtree/include/pcl/kdtree/kdtree_flann.h

```diff
@@ -207,10 +207,10 @@ namespace pcl
       getName () const override { return ("KdTreeFLANN"); }
 
       /** \brief A FLANN index object. */
-      std::shared_ptr<FLANNIndex> flann_index_;
+      shared_ptr<FLANNIndex> flann_index_;
 
-      /** \brief Internal pointer to data. TODO: replace with std::shared_ptr<float[]> with C++17*/
-      std::shared_ptr<float> cloud_;
+      /** \brief Internal pointer to data. TODO: replace with shared_ptr<float[]> with C++17*/
+      shared_ptr<float> cloud_;
```

Comment:
Your changing to boost::shared_ptr here again.

Reply:
Als flagged by @kunaltyagi, fixed.